src/kernel/include/tcap.h

```diff
@@ -269,4 +269,14 @@ tcap_higher_prio(struct tcap *a, struct tcap *c)
 	return ret;
 }
 
+static inline int
+tcap_introspect(struct tcap *t, unsigned long op, unsigned long *retval)
+{
+	switch(op) {
+	case TCAP_GET_BUDGET: *retval = t->budget.cycles; break;
+	default: return -EINVAL;
```

Comment:
if you're putting these all on the same line, then at least line up the "bodies" of each case/default so that it is easier to visually parse what's happening.